xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1149 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE chapter [
+<!ENTITY % entities SYSTEM "entity-decl.ent">
+%entities;
+]>
+<chapter version="5.1" xml:id="cha.obs.scm_ci_workflow_integration"
+         xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         xmlns:ns7="http://docbook.org/ns/transclusion"
+         xmlns:ns6="http://www.w3.org/1998/Math/MathML"
+         xmlns:ns5="http://www.w3.org/2000/svg"
+         xmlns:ns4="http://www.w3.org/1999/xhtml"
+         xmlns:ns="http://docbook.org/ns/docbook">
+  <title>SCM/CI Workflow Integration [Beta Feature]</title>
+
+  <sect1 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup">
+    <title>SCM/CI Workflow Integration Setup</title>
+
+    <note>
+      <title>The feature described in this chapter is a <emphasis>Beta
+      Feature</emphasis>.</title>
+
+      <para>That means the development of this feature is still in progress and under
+      constant revision. We can substantially change this feature throughout the process
+      or even discard instances. Make sure to check back before reporting problems.
+      </para>
+
+      <para>Everybody can join the beta program, read <link
+      xlink:href="https://openbuildservice.org/2018/10/04/the-beta-program">here</link>
+      how to do it.</para>
+    </note>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.introduction">
+      <title>Introduction</title>
+
+      <para>With this integration you can take advantage of source code management (SCM)
```

Comment:
```suggestion
      <para>With this integration, you can take advantage of source code management (SCM)
```

Reply:
I think this coma is not really needed...

Reply:
For shorter sentences, it's not always needed to have a [comma for prepositional phrases](https://prowritingaid.com/grammar/1000157/Why-should-you-put-comma-after-prepositional-phrases). For longer sentences, it is though.

I prefer to always have a comma since it's correct for short and long sentences. This way, I don't need to check if my sentences are short enough. It's just simpler.

Reply:
Okay, I'll commit this suggestion, then